src/MassTransit/Transports/TransportLogMessages.cs

```diff
@@ -7,16 +7,16 @@ namespace MassTransit.Transports
 
     public static class TransportLogMessages
     {
-        public static readonly LogMessage<string> ConnectHost = LogContext.Define<string>(LogLevel.Debug,
+        public static readonly LogMessage<string> ConnectHost = LogContext.Define<string>(LogLevel.Information,
             "Connect: {Host}");
 
-        public static readonly LogMessage<string> ConnectedHost = LogContext.Define<string>(LogLevel.Debug,
+        public static readonly LogMessage<string> ConnectedHost = LogContext.Define<string>(LogLevel.Information,
             "Connected: {Host}");
 
-        public static readonly LogMessage<string> DisconnectHost = LogContext.Define<string>(LogLevel.Debug,
```

Comment:
Disconnecting is not an error, but it could be informational.

Reply:
Disconnecting is very Important,why not warning ?

Reply:
Because it's normal operation, informational is fine. If there are errors connecting, that's different.